### PR TITLE
Initial commit for the Schedule Meeting Modal Window

### PR DESCRIPTION
**Pull request by colummurphy**
I modified the DateComponent to add the ability to highlight an array of dates.  I added a public route to the App component to
allow the modal window to be displayed.  I added an error function to the App component to handle server errors.  Seven files were added for the Modal Window function.

### CODE REVIEW

File: src/component/scheduleMeetingModal/AvailableTimeList.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+    <div className='itemsContainer'>
+      { timeSlotsForDate.map(timeSlot => {
+        return (
+          <div 
+            key={timeSlot.id}
+            className='itemStyle' 
+            style={{ backgroundColor: (selectedTimeSlot !== null 
+              && selectedTimeSlot.id === timeSlot.id) ? 'yellow' : 'white' }}
```
Comment (allinthemind66) [inline src/component/scheduleMeetingModal/AvailableTimeList.js]: maybe have these strings as constants, but not a blocker

File: src/component/scheduleMeetingModal/AvailableTimeList.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          >
+            <div>
+              {formatTime(timeSlot.startDateTime, timeSlot.endDateTime)}
+            </div>              
+            <BsPlusCircleFill
+              id='addIcon'
+              style={{color: 'blue', cursor: 'pointer'}}
```
Comment (allinthemind66) [inline src/component/scheduleMeetingModal/AvailableTimeList.js]: maybe have this blue color as a constant, but not a blocker

File: src/component/scheduleMeetingModal/AvailableTimeList.js (hunk 1 above)
Comment (colummurphy) [reply src/component/scheduleMeetingModal/AvailableTimeList.js]: Added const for component colors

File: src/component/scheduleMeetingModal/AvailableTimeList.js (hunk 2 above)
Comment (colummurphy) [reply src/component/scheduleMeetingModal/AvailableTimeList.js]: Added const for component colors

**Pull request merged**